_problems/post-subsidy-security.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: problem
+title:  "Post-subsidy Security"
+tags: consenus, mining, incentives, PoW
+status: "open"
+maintainer: VzxPLnHqr
+issue: 29
+---
+
+As the block subsidy drops, the network security budget may be inadequate if a robust fee market fails to develop. Two forms of the problem are overall low fees, and adequate-but-bursty fees. In both cases the network may fail to garner enough work to properly secure itself.
+
+A goal of this research is to define one or more *soft forks* which either permanently
+solve or, at minimum, stave off the problem, without sacrificing current community values.
+This means, for example, a hard-fork which introduces inflation will not be considered here.
+
+## Impact
+
+<!-- - Try not to repeat the description too much -->
+<!-- - Make it clear what the impact on the big picture of Bitcoin's evolution would be -->
+
+Ultimately a reduced security, if sustained for long enough, manifests as reduced confidence in the future of the network.
+Reduced confidence could trigger a further reduction in security, and a downward spiral ensues. If the 
+network cannot offer an attractive enough reward to those contributing to its security (miners), then this threat, while far
+off right now (2022), could become an existential threat. 
+
+
+## Potential Directions
+
+<!-- - The main use of listing hand-wavy directions is useful to further explore the problem. -->
+
+First we need to clearly define what a "security budget" for bitcoin is in such a way that nodes can 
+measure it endogenously in an incentive compatible way (e.g. without reference/reliance on something 
+outside the protocol like USD/tx). If nodes cannot measure it, it is exceedingly difficult to design
+a fix.
+
+
+## Proposed Solutions
+
+### Defining the Security Budget
+
+Some possible (currently quite informal) definitions for security budget and associated solutions: 
+
+1. Sztorc claims this can be solved in [1] via merged-mining and proposes a per-block security budget 
+   `SB` as `SB = P*Q` where `P` is the average transaction price paid in sats/vbyte, and `Q` is the 
+   number of bytes sold, the block size, in bytes/block. 
+   
+2. ...
+   
+ The above are just example definitions and informal possible directions. Yet, if there is to be a formally
+ defined notion of a security budget for the network, then we also should be asking what the appropriate "target"
+ for that budget is. We probably seek some goldilocks scenario here where the amount of security garnered by
+ the network is "not too much," definitely "not too little," but "just right" (in expectation of course).
```

Comment:
I think you could define an adequate security "target" in function of a threat model, where you encompass attacker's available resources and capabilities.

See also recent literature (not parsed yet) seeking for economic limits of Bitcoin: https://ericbudish.org/wp-content/uploads/2018/06/Economic-Limits-Bitcoin-Blockchain-June-2022.pdf